packages/scene-composer/src/assets/auto-gen/icons/Light.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import { SVGProps } from 'react';
```

Comment:
These are auto-generated, tehy shouldn't be included in the CR. Check your `.gitignore` file.